src/main/java/eu/europeana/metis/sandbox/controller/DatasetController.java

```diff
@@ -54,6 +59,7 @@
   private final ZipService zipService;
   private final DatasetService datasetService;
   private final DatasetReportService reportService;
+  private DatasetController test;
```

Comment:
I think this is some leftovers?

Reply:
You are correct, it is some leftovers, will remove it